stable/grafana/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: grafana
-version: 0.8.5
+version: 0.9.0
```

Comment:
Nitpicky: I would argue 1.0.0 is more appropriate here, since this is a backwards incompatible change

Reply:
Fixed.